src/pages/writing-nft/index.vue

```diff
@@ -150,6 +190,29 @@ export default {
         // No-op
       }
     }
+    const [trendingRes, latestRes] = await Promise.all([
+      this.$axios.$get(getTopNFTClasses()),
+      this.$axios.$get(getLatestNFTClasses()),
+    ]);
+    [this.trendingClassIds, this.latestClassIds] = [trendingRes, latestRes].map(
+      res =>
+        (res.classes || [])
+          .filter(
+            c => c.metadata?.nft_meta_collection_id === 'likerland_writing_nft'
+          )
+          .map(c => c.id)
+          .slice(0, 10)
+    );
+  },
+  methods: {
+    handleTabClick(tab) {
+      this.currentTab = tab;
```

Comment:
I think `currentTab` should be a computed prop instead of data, as we can count on `tab` in qs